### PR TITLE
update for rails 5

### PR DESCRIPTION
**Pull request by xiaopow**

### CODE REVIEW

**Comment by dancinglightning** [comment]
Just had a look to see what this is about, because i am running 1.5.2 quite fine with rails 5.

So i guess this not needed, yes?

**Comment by dtelaroli** [comment]
:+1:

File: impressionist2.gemspec [hunk 1]
```diff
@@ -3,7 +3,7 @@ $:.push File.expand_path('../lib', __FILE__)
 require 'impressionist/version'
 
 Gem::Specification.new do |s|
-  s.name          = 'impressionist'
+  s.name          = 'impressionist2'
```
Comment (jgrau) [inline impressionist2.gemspec]: What's the reason for this change?

**Comment by jgrau** [comment]
Thanks for the PR @xiaopow. I realize this is an old PR but I would love to get it merged.

**Comment by jgrau** [comment]
I am closing this as stale. We will get rails 5 support merged in soon with backwards compatibility for rails 3.x. Please feel free to reopen if you feel the need.
